examples/tv-app/android/App/app/src/main/java/com/tcl/chip/chiptvserver/service/MatterServant.java

```diff
@@ -35,12 +35,15 @@
 import com.tcl.chip.tvapp.LowPowerManagerStub;
 import com.tcl.chip.tvapp.MediaInputManagerStub;
 import com.tcl.chip.tvapp.MediaPlaybackManagerStub;
+import com.tcl.chip.tvapp.OnOffManagerStub;
 import com.tcl.chip.tvapp.TvApp;
 import com.tcl.chip.tvapp.WakeOnLanManagerStub;
 
 public class MatterServant {
 
   private ChipAppServer chipAppServer;
+  private TvApp mTvApp;
+  private boolean mOnOff = true;
```

Comment:
How about saying `mIsOn` or `mIsOff`? saying "onoff is true" does not seem as obvious as "ison is true".

Reply:
fixed